### ISSUE TITLE
xsuite-prebuild fails due to kernel_definitions not included in release

### ISSUE THREAD

**Issue by eltos** [xsuite/xsuite#273]
Thanks a lot for providing the option of precompiling kernels. Unfortunately, the command `xsuite-prebuild` fails with the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/xsuite-prebuild", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/xsuite/cli.py", line 21, in main
    regenerate_kernels()
  File "/usr/local/lib/python3.10/dist-packages/xtrack/prebuild_kernels.py", line 175, in regenerate_kernels
    with metadata_file.open('r') as fd:
  File "/usr/lib/python3.10/pathlib.py", line 1119, in open
    return self._accessor.open(self, mode, buffering, encoding, errors,
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.10/dist-packages/xtrack/prebuilt_kernels/_kernel_definitions.json'
```

- [ ] And indeed the kernel definitions are missing after a fresh installation with pip.
  In fact, the file/folder is not even in the tar one can download from https://pypi.org/project/xtrack/0.25.0/#files.
  After manual download of the files from GitHub it works.

- [ ] BTW there is a typo in the docs at https://xsuite.readthedocs.io/en/latest/installation.html: the command is `xsuite-prebuild`, not `xsuite-precompile` as documented

/cc @szymonlopaciuk

**Comment by giadarol** [comment]
Concerned files should be added in 
https://github.com/xsuite/xtrack/blob/main/MANIFEST.in

**Comment by szymonlopaciuk** [comment]
I am currently working on the documentation of xsuite, so I will shortly address the second point as well.

**Comment by giadarol** [comment]
solved in xtrack v0.26.0
